atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -280,6 +282,19 @@ public void setsGlobalDefaultLockTimeout() {
         assertEquals(expectedTimeout, lockRequest.getLockTimeout());
     }
 
+    // TODO (tpetracca): remove this test by November 15th, 2017
+    @Test
+    public void canCreateUsingDeprecatedMethods() {
+        AtlasDbConfig realConfig = ImmutableAtlasDbConfig.builder()
+                .keyValueService(new InMemoryAtlasDbConfig())
+                .defaultLockTimeoutSeconds(62)
+                .build();
+
+        Supplier<Optional<AtlasDbRuntimeConfig>> configSupplier = () -> Optional.of(runtimeConfig);
+        TransactionManagers.Environment env = mock(TransactionManagers.Environment.class);
+        TransactionManagers.create(realConfig, configSupplier, ImmutableSet.of(), env, false);
```

Comment:
I think we need to test *every* incarnation of `.create(...)`, since some of them land on different builders in the end and they're all public API.

Reply:
Good idea - I just lazily tested with the base one, because I could see that the others all work, and because of time pressures. I can go back and add the others :)

Reply:
See #2595.